src/test/java/com/bringframework/component/TestComponent.java

```diff
@@ -0,0 +1,13 @@
+package com.bringframework.component;
```

Comment:
suggest to  move `TestComponent` and `TestService` to `com.bringframework.testinfrastructure.component`

Reply:
I'm sorry, but I don't understand why? and what is the reason for doing it? Could you explain why "test infrastructure" is needed here? or why it would be better than just having the same package structure as in src?

Reply:
moved to `reader`